projects/core/components/button/button.style.less

```diff
@@ -119,11 +119,19 @@
 .t-icon {
     :host[tuiIconButton] & {
         display: flex;
-        width: 100%;
-        height: 100%;
         align-items: center;
         justify-content: center;
     }
+
+    :host[tuiIconButton][data-size='s'] &,
+    :host[tuiIconButton][data-size='xs'] &,
+    :host[tuiIconButton][data-size='l'] &,
+    :host[tuiIconButton][data-size='m'] & {
+        width: 100%;
+        height: 100%;
+        max-width: 1.5rem;
```

Comment:
Icons should be 1rem for `s` and `xs` sizes for both `tuiButton` and `tuiIconButton`.

Reply:
I have a problem then
![image](https://user-images.githubusercontent.com/12021443/231963732-5fdb8b59-5cff-4421-a26d-5c4ed4a055de.png)
![image](https://user-images.githubusercontent.com/12021443/231963773-fe261844-3df8-4f6c-b328-b346971248d2.png)


I think it's breaking changes